### DISCUSSION TITLE
Support progress reporting

### DISCUSSION THREAD

**Discussion by Jan200101** [ghostty-org/ghostty#3123] [Feature Requests, Ideas] [answered]
Some linux applications (namely flatpak and [selinux](https://mastodon.social/@pid_eins/113406672373007116) with pull-requests existing for other [cargo](https://github.com/rust-lang/cargo/pull/14615) and possibly other applications) have started to implement progress reporting via the ConEmu `OSC 9;4` command

In essence this would allow a program to convey its progress in a way that wouldn't need the terminal active

How to exactly implement progress reports is up to debate, some options are:
- include progress in tab and or window title
- set progress on the taskbar

What other terminals do:
- alacritty: doesn't support it, [no explanation why](https://github.com/alacritty/alacritty/issues/5201) 

- kitty & foot: [check for `OSC 9;4` specifically and drop it entirely](https://github.com/kovidgoyal/kitty/commit/847433dba2271031cd7d58025022eef10a668072)

- windows terminal: shows progress as a circle in the tab ([added by this PR](https://github.com/microsoft/terminal/pull/8055))
![image](https://github.com/user-attachments/assets/6a19712b-8ebc-498e-b1b0-74a628db8c01)

- conemu (where this originated): shows a progress number in the tab title    
![image](https://github.com/user-attachments/assets/906e4400-0c2f-4c5d-b05b-52dc7bdfedfb)

- Gnome VTE library (Gnome Terminal, guake, etc.): [implemented](https://gitlab.gnome.org/GNOME/vte/-/commit/a73b309ee992f31923f148045130bc41e55a3303)

- ptyxis: shows progress as a circle in the tab
![image](https://github.com/user-attachments/assets/23c39f4e-0c94-4dd6-9487-118d88f6539a)

**Comment by rhodes-b** [comment, marked as answer]
https://github.com/ghostty-org/ghostty/pull/7975

this is solved?
